Manifest.toml

```diff
@@ -0,0 +1,7 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Please remove this file from the PR.

Reply:
Sure, hence the .gitignore ;)

As I've indicated, I plan to squash this all into one commit when you're otherwise ready for a merge. it won't include the Manifest.